test/e2e/kubectl.go

```diff
@@ -186,9 +220,14 @@ func runKubectl(args ...string) string {
 	if testContext.provider == "gke" {
 		kubectlArgs = append(kubectlArgs, "--server="+testContext.host)
 	}
-	Logf("Running 'kubectl %v'", strings.Join(kubectlArgs, " "))
 	cmd := exec.Command("kubectl", kubectlArgs...)
+	Logf("Running '%s %s'", cmd.Path, strings.Join(cmd.Args, " "))
```

Comment:
Then I'll need to duplicate it in BeforeEach as well (since that also runs kubectl). I put it here to try and avoid the code duplication.